src/test/java/org/fundacionjala/coding/abel/CamelCaseTest.java

```diff
@@ -0,0 +1,79 @@
+package org.fundacionjala.coding.abel;
+
+import static org.junit.Assert.assertEquals;
+
+
+import org.junit.Test;
+
+/**
+ *
+ */
+public class CamelCaseTest {
+
+
+    /**
+     *
+     */
+    @Test
+    public void testTwoWords() {
+        CamelCase c = new CamelCase();
+        assertEquals("TestCase", c.camelCase("test case"));
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void testThreeWords() {
+        CamelCase c = new CamelCase();
```

Comment:
You are repeating this line of code in all your unit tests
Please move it to an @Before method.